assets/css/style.css

```diff
@@ -0,0 +1,83 @@
+@media only screen and (max-width: 768px) {
+  body {
+    overflow: hidden;
+  }
+
+  .navbar {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+  }
+
+  .nav-branding {
+    color: #28352f;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 6vw;
+    line-height: 2.8rem;
+    text-decoration: none;
+    margin-left: 2.4vh;
+    margin-top: 5.4vh;
+  }
+
+  .humburger {
+    display: block;
+    margin-right: 1vh;
+    margin-top: 5.4vh;
+  }
+
+  .humburger .bar {
+    display: block;
+    width: 25px;
+    height: 3px;
+    color: brown;
+    margin: 5px auto;
+    -webkit-transition: all 0.3s ease-in-out;
+    transition: all 0.3s ease-in-out;
+    background-color: #28352f;
+  }
+
+  .headline {
+    position: relative;
+    background-image: url(/assets/images/header-bg.png);
+    background-repeat: no-repeat;
+    background-size: 90%;
+    height: 67.15vh;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    justify-items: center;
+    margin-left: 2.4vh;
+    text-align: center;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 5vw;
+    line-height: 3rem;
+    color: #172b4d;
+  }
+
+  .greenTxt {
+    display: block;
+    color: #36b37f;
+  }
+
+  p {
+    font-family: "Inter", sans-serif;
```

Comment:
- [ ]  If you want to create a CSS rule for multiple elements, kindly use a class attribute instead. For example, instead of using `font-family: "Inter", sans-serif;` 3 times , you can create a class and add it to all elements that needs that font  example : 
`.inter-font {font-family: "Inter", sans-serif;}`

Reply:
.inter-font {font-family: "Inter", sans-serif;} 
added this class to style file